tests/framework/extensions/kubectl/apply.go

```diff
@@ -0,0 +1,77 @@
+package kubectl
```

Comment:
are these part of the CRD generation? if not, I think you could move these to a separate PR.